Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:alpine
+FROM conex.eff.org/techops/nginx-base:latest
```

Comment:
Like we mentioned, let's add a comment here about how this is for internal use only.

Reply:
added!